sdk/synapse/synapse-artifacts/README.md

```diff
@@ -45,6 +48,9 @@ for await (const pipeline of pipelinesByWorkspace) {
 Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
 
 ```ts snippet:SetLogLevel
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
@kevinzz6  why we have this? manually added?

Reply:
@MaryGao I ran rush build -t . and generate this code change.

Reply:
@kazrael2119 do you know why we added the license header here?